QuickOSM/ui/dialog.py

```diff
@@ -132,6 +134,12 @@ def __init__(self, iface=None, parent=None):
             Panels.QuickQuery: self.action_xml_qq,
             Panels.Query: self.action_xml_q,
         }
+        icon = QIcon(resources_path('icons', 'QuickOSM.svg'))
+        self.reload_action = QAction(icon, tr("Reload the query in a new file"), self.iface)
+        actions = Actions(self)
+        reloader = partial(actions.pre_run_reload)
+        self.reload_action.triggered.connect(reloader)
```

Comment:
I think you can skip the `partial`, no ?

Reply:
When I did suppress the partial, the action didn't connect with the function so when I clicked on the button nothing would happen.

Reply:
Something strange here. But not a blocker.
`partial` shouldn't be needed.